libs/deps/plans-modal.js

```diff
@@ -4232,6 +4232,7 @@ var setCustomTemplateLiteralTag = (e16) => {
 
 // ../node_modules/@spectrum-web-components/icons-ui/src/icons/Cross200.js
 var Cross200Icon = ({ width: t13 = 24, height: e16 = 24, title: r9 = "Cross200" } = {}) => tag`<svg
+    aria-hidden="true"
```

Comment:
These SVGs already have the attribute, no updates required.

Reply:
Removed it from here 👍